src/externalized/content/NewStrings.cc

```diff
@@ -611,3 +615,107 @@ const ST::string GetModifiersForLockBlowUp(SOLDIERTYPE* const s)
 
 	return result;
 }
+
+const ST::string GetModifiersForEffectiveAttributes(SOLDIERTYPE* const s, Attributes const attrName)
+{
+	ST::string result{}, comment{}, possession, location;
```

Comment:
You don't need the {} after the definition of a ST::string, like any proper C++ object it has a default constructor that properly initializes itself.